licenses/spdx/BUILD

```diff
@@ -2506,3 +2506,429 @@ license_kind(
     conditions = [],
     url = "https://spdx.org/licenses/zlib-acknowledgement.html",
 )
+
```

Comment:
That would be a great idea. It is just a little harder to code.
I opted for the simple case right for now (append only) because I want a path where any existing user edits like adding conditions are preserved.
My prioritization is always get to MVP first, then improve the polish.

#20

Reply:
Sounds good 👍